extensions/quit-application/package.json

```
@@ -0,0 +1,42 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "quit-application",
+  "title": "Quit Application",
```

I would suggest rename it to plural, since it actually allows you to quite multiple applications within the same command
```suggestion
  "title": "Quit Applications",
```